packages/plugin-php/__tests__/index.js

```diff
@@ -0,0 +1,28 @@
+import { PHP_FUNC } from '@octolinker/helper-grammar-regex-collection';
+import php from '../index';
+
+describe('rust-crate', () => {
```

Comment:
Rename to something php related?

```suggestion
describe('php', () => {
```